Makefile

```diff
@@ -42,6 +42,14 @@ staging: test-cluster
 		$(eval export TF_VAR_environment=${ENVIRONMENT})
 		$(eval include global_config/staging.sh)
 
+
```

Comment:
I think the convention is a single empty newline between commands.

Reply:
double space removed